src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -118,12 +120,18 @@ export const ClaimSelect: FC<ClaimSelectProps & ClaimRouteProps> = ({
 
   const claimBody = useMemo(() => {
     if (!stakingAssetAccountAddress) return <ChainNotSupported chainId={stakingAsset?.chainId} />
-    if (isUnstakingRequestLoading || isUnstakingRequestPending || isUnstakingRequestPaused)
+    if (
+      isUnstakingRequestLoading ||
+      isUnstakingRequestPending ||
+      isUnstakingRequestPaused ||
+      isUnstakingRequestRefetching ||
+      !isUnstakingRequestEnabled
+    )
       return new Array(2).fill(null).map(() => <Skeleton height={16} my={2} />)
-    if (isUnstakingRequestError || !unstakingRequestResponse.length)
+    if (isUnstakingRequestError || (isUnstakingRequestSuccess && !unstakingRequestResponse.length))
       return <NoClaimsAvailable isError={isUnstakingRequestError} setStepIndex={setStepIndex} />
 
-    return unstakingRequestResponse.map((unstakingRequest, index) => {
+    return unstakingRequestResponse?.map((unstakingRequest, index) => {
```

Comment:
Unfortunately, we lost type inference of `unstakingRequestResponse` being defined here, but it effectively will

Reply:
Disregard me - the fix was absolutely wrong and after way too many debugger runs, the actual fix keeps type inference

7ac50c607cc684eaca24aa9f7055c76304cc3735